MobileCenter.xcworkspace/xcshareddata/xcschemes/AllLibraries for macOS.xcscheme

```diff
@@ -48,6 +48,48 @@
                ReferencedContainer = "container:MobileCenterCrashes/MobileCenterCrashes.xcodeproj">
```

Comment:
I think schemes should all following the same naming conventions, for instance this one should be `AllLibrariesMacOS` and the iOS one would be `AllLibrariesIOS`. I think there's a couple of other schemes like that.